docs/source/publications.rst

```diff
@@ -30,15 +30,20 @@ Hill, J., Rush, G., Peakall, J., Johnson, M., Hodson, L., Barlow, N.L.M.,
 Bowman, E.T., Gehrels, W.R., Hodgson, D.M., Kesserwani, G:
 Resolving tsunami wave dynamics: integrating sedimentology and numerical modelling.
 *The Depositional Record*
-doi: `10.1002/dep2.247 <https://doi.org/10.1002/dep2.247`>_, 2023.
+doi: `10.1002/dep2.247 <https://doi.org/10.1002/dep2.247>`_, 2023.
+
+Zhang, C., Zhang, J., Angeloudis, A., Zhou, Y., Kramer, S.C., Piggott, M.D.:
```

Comment:
I am not sure this paper used any thetis simulations... are we sure we want to list this here?

Reply:
Ah yes, you're quite right - that one shouldn't have been included

Reply:
I plan with @cpjordan to add some changes to some examples  shortly, maybe I can try to add a few more then and correct this